.drone.yml

```diff
@@ -449,6 +457,7 @@ steps:
   - cd /hugo && make prod
   depends_on:
   - initialize
+  - build-frontend-docs
```

Comment:
I think the build-frontend-docs step is missing for the master pipeline.

Reply:
There are no need to build that one on the master build so I just skipped it. But do you think we should build it there as well? We are running it in the `grafana -> website` sync.

Reply:
Great catch @aknuds1 !